finance-api/api-reference.md

```diff
@@ -0,0 +1,288 @@
+|CASHBACK_PAYOUT (Deprecated) |No available description.
+|EMONEY_TRANSFER (Deprecated) |No available description.
+|FAILED_PAYOUT |A previous payout transaction has failed and been made void. The payout money is returned to the merchant's Zettle liquid account.
+|FEE_DISCOUNT_REVOCATION |An internal reclaim of an outstanding fee discount that is not consumed within a certain time frame. As these funds are reclaimed from a special fee discount account, the transaction will not be visible in the Zettle liquid account of the merchant.
+|FROZEN_FUNDS |The money that is frozen to cover a chargeback. When the issuing back initiates a chargeback, the money will be removed from the merchant's liquid account and marked as frozen to cover the chargeback. If the chargeback is later revoked, the money will be returned to the merchants liquid account with a new and positive transaction of the same type. It effectively makes the initial FROZEN_FUNDS transaction void.
+|INVOICE_PAYMENT |An invoice payment.
+|INVOICE_PAYMENT_FEE |An invoice payment fee.
+|PAYMENT |An alternative third-party payment method where Zettle handles the funds. For example, PayPal QR code and Klarna QR code.
+|PAYMENT_FEE |The fee for a third-party payment method. For example,PayPal QR code and Klarna QR code.
+|PAYOUT |A payout from the merchant's Zettle liquid account to the merchant’s bank account. If the merchant is a PayPal user, the payout will be made to their PayPal Wallet.  
+|TELL_FRIEND (Deprecated) |No available description.
+|VOUCHER_ACTIVATION |Used when activating a voucher. The money is inserted to the merchant’s fee discount account instead of preliminary and liquid accounts.
+ 
+</details>
+
+
+### Responses
+<details open="true">
+<summary>HTTP status codes</summary>
+
+|Status code |Description
+|:---- |:----
+|200 OK |Returned when the account transactions are successfully fetched.  
+|403 Forbidden |Returned when you are not authorised to fetch the account transactions.
+|400 Bad Request |Returned when a required parameter is missing or in a wrong format in the request.  
+</details>
+
+<details open="true">
+<summary>Response attributes</summary>
+
+|Name |Type |Description
+|:---- |:---- |:----
+|data |an array of objects|A list of transactions for the given account in the descending order that shows the most recent transaction first.
+|timestamp |string |Time when a transaction happens in the merchant's Zettle account. It's not the timestamp when a card transaction or a purchase happens. <br/>For example, transactions in the merchant's liquid account may happen hours after card transactions or purchases take place.
+|amount |integer |The amount of money of a transaction. If the transaction is a refund or a payment fee, the amount is negative.
+|originatorTransactionType |string |The transaction type. See [supported transaction types](#supported-transaction-types). 
+|originatingTransactionUuid |string |The transaction identifier as UUID version 1.  
```

Comment:
This description makes me think that  `originatingTransactionUuid` is a unique identifier of this particular transaction event. Is it though?

```suggestion
|originatingTransactionUuid |string |The identifier of the originating transaction as UUID version 1.  
```